.github/workflows/test.yml

```diff
@@ -69,58 +40,60 @@ jobs:
       run: |
         cargo clippy -- -D warnings
 
-  release-linux:
+  test-ubuntu:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust: [stable, nightly]
     steps:
     - name: Checkout repository
       uses: actions/checkout@v2
     - name: Install Rust
       uses: actions-rs/toolchain@v1
       with:
-        toolchain: stable
+        toolchain: ${{ matrix.rust }}
         override: true
         profile: minimal
-    - name: Check Rust Version
+    - name: Build
       run: |
-        ci/check_version.sh
```

Comment:
the check_version needs to be run each time, thus it is more suited for the test than release